data_structures/src/data_request.rs

```diff
@@ -514,7 +518,14 @@ pub fn calculate_reward_collateral_ratio(
     };
 
     if witness_reward > 0 {
-        dr_collateral / witness_reward
+        let ratio = dr_collateral / witness_reward;
+
+        if dr_collateral % witness_reward == 0 {
+            ratio
+        } else {
+            // If the ratio is not an integer, e.g. 123.0001, we round it to the next integer, e.g. 124.
+            ratio.saturating_add(1)
+        }
```

Comment:
Can't we simply `dr_collateral / witness_reward).round()`?

If not, i hope we can have [`div_ceil`](https://doc.rust-lang.org/std/primitive.u64.html#method.div_ceil) soon...